### DISCUSSION TITLE
Eloquent: API Resources

### DISCUSSION THREAD

**Discussion by ghost** [adonisjs/core#3932] [Ideas]
Hi,
is there something like Eloquent: API Resources? https://laravel.com/docs/9.x/eloquent-resources

**Comment by McSneaky** [comment]
Sounds like serializing models: https://docs.adonisjs.com/guides/models/serializing-models

**Comment by ghost** [comment]
I think I found it:
https://github.com/Thanatad/ts-rc-transforms
https://github.com/kmorpex/adonis-bublebee-ts

Adonisjs should add something like that it's very helpful in Laravel
